docs/root/configuration/http/http_filters/aws_lambda_filter.rst

```diff
@@ -0,0 +1,132 @@
+
+.. _config_http_filters_aws_lambda:
+
+AWS Lambda
+==========
+
+* :ref:`v2 API reference <envoy_api_msg_config.filter.http.aws_lambda.v2alpha.config>`
+* This filter should be configured with the name *envoy.filters.http.aws_lambda*.
+
+.. attention::
+
+  The AWS Lambda filter is currently under active development.
+
+The HTTP AWS Lambda filter is used to trigger an AWS Lambda function from a standard HTTP/1.x or HTTP/2 request.
+It supports a few options to control whether to pass through the HTTP request payload as is or to wrap it in a JSON
+schema.
+
+If ``payload_passthrough`` is set to ``true``, then the payload is sent to Lambda without any transformations.
```

Comment:
Please ref link all config options (or at least the first one)

Reply:
Sure.